dbms/src/Flash/Mpp/GRPCReceiverContext.h

```diff
@@ -28,7 +28,8 @@
 namespace DB
 {
 using MPPDataPacket = mpp::MPPDataPacket;
-using MPPDataPacketPtr = std::shared_ptr<MPPDataPacket>;
+using MPPDataPacketPtr = std::shared_ptr<DB::TrackedMppDataPacket>;
```

Comment:
Why use `MPPDataPacketPtr` as an alias of `TrackedMppDataPacketPtr`, it may make the reader confusing.

Reply:
good spot